regnety/models/utils.py

```diff
@@ -0,0 +1,37 @@
+import tensorflow as tf
```

Comment:
I assume this is written to parallel the behavior of the original regnet implementation. It would be great to add some module level documentation explaining this and referencing the original code.

Same for the _pca_jitter and _inception_style_crop functions from the other file.

Reply:
> I assume this is written to parallel the behavior of the original regnet implementation.


No, that is unintentional. I didn't find a better place to keep the initializers, so I made a utils file.  

I'll add the documentation.

Reply:
Added module level docstrings for important modules.

Reply:
Sounds good. I'm not saying it looks the same (I haven't looked) I just know you were looking at that implementation to try to resolve the accuracy discrepancy.